css/style.css

```diff
@@ -1,61 +1,281 @@
-.banner {
-width: 1600px;
-height: 392px;
-background: #000000;
-opacity: 0.3;
-background-image: url(/images/banner\ image.jpg);
-}
-
-.titlebanner {
-width: 566px;
-height: 42px;
-font-style: normal;
-font-weight: 500;
-font-size: 36px;
-line-height: 42px;
-color: #FFFFFF;
-}
-
-.btn{
-left: 24.23%;
-right: 24.62%;
-top: 33.33%;
-bottom: 32.1%;
-font-style: normal;
-font-weight: 500;
-font-size: 24px;
-line-height: 28px;
-color: #FFFFFF;
-}
-
-.title {
-font-style: normal;
-font-weight: bold;
-font-size: 18px;
-line-height: 21px;
-/* identical to box height */
-color: #000000;
-}
-
-.author{
-font-style: italic;
-font-weight: normal;
-font-size: 18px;
-line-height: 21px;
-/* identical to box height */
-color: #000000;
-}
-
-.books {
- display: flex;
- flex-wrap: wrap;
-}
- 
-.book {
-    width: 20%;
-    /* box-sizing: border-box; */
-    margin: 0 20px;
-}
+          /* General Styling */
 
+          *{
+            margin:0;
+            padding:0;
+        }
+        
+                  /* Top section styling */
+        
+        .top-section{
+            height:100px;
+            display: flex;
+            align-items: center;
+        }
+        
+        .top-section h1{
+            font-weight:500;
+            font-family: 'Vollkorn', serif;
+            font-size: 48px;
+            margin-left:4rem;
+            
+        }
+        
+        
+            /* Image section styling */
+        
+        .img-section{
```

Comment:
the same image always used. image should be changed according to the media query as well.

Reply:
Background Image is being changed according to screen size , I don't know if you noticed that I was changing 'background-size' property from cover to contain and vice versa in media query lines of code.
And about image's name , it was already named with a space between banner and image after I cloned it to my laptop so I though that you are ok with it since you were with Pourija when he was writing html and css during yesterday's class.

Reply:
Image is resized but what if you have image that is for very large screen and really heavy and I have slow internet and low resolution screen. I will still get the big image resized to a certain resolution but only on screen.
i.e. I have 2560x1440 and get the image that is 2 mb size, I have 640x480 resolution and I get the same 2 mb image.
I should get image that is appropriate for another resolution.